net-misc/gnome-remote-desktop/gnome-remote-desktop-43.0.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+inherit gnome.org gnome2-utils meson python-any-r1 systemd xdg
+
+DESCRIPTION="Remote desktop daemon for GNOME using pipewire"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/gnome-remote-desktop"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc +rdp systemd +vnc aac"
+REQUIRED_USE="|| ( rdp vnc ) aac? ( rdp )"
+RESTRICT="test" # Tests run xvfb-run directly
+
+DEPEND="
+	x11-libs/cairo
+	>=dev-libs/glib-2.68:2
+	>=media-video/pipewire-0.3.49:=
+	app-crypt/libsecret
+	x11-libs/libnotify
+	>=media-libs/libepoxy-1.4
+	x11-libs/libdrm
+	app-crypt/tpm2-tss
+	rdp? (
+		>=media-libs/nv-codec-headers-11.1.5.0
+		>=net-misc/freerdp-2.8.0:=[server]
+		>=sys-fs/fuse-3.9.1
+		>=x11-libs/libxkbcommon-1.0.0
+	)
+	vnc? (
+		net-libs/libvncserver
+	)
+	aac? (
+		media-libs/fdk-aac
+	)
```

Comment:
Since the `REQUIRED_USE` is in place, it doesn't affect anything, but we should put the `aac? ( ... )` block inside the `rdp? ( ... )` block to match meson.build.